sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -0,0 +1,209 @@
+
+    if (name == offButtonLocalizedTitle) {
+      this.props.onSelectClosedCaptions("");
+    } else if (this.props.selectedClosedCaptionsLanguage !== name) {
+      this.props.onSelectClosedCaptions(name);
+    }
+  },
+
+  onDismissPress: function() {
+    this.props.onDismiss();
+  },
+
+  renderHeaderView: function(hasMultiAudioTracks, hasClosedCaptions) {
+    var leftTitle;
+    var rightTitle;
+
+    var localizedAudioTitle = Utils.localizedString(this.props.config.locale, "Audio", this.props.config.localizableStrings);
+    var localizedSubtitlesTitle = Utils.localizedString(this.props.config.locale, "Subtitles", this.props.config.localizableStrings);
+
+    if (hasMultiAudioTracks && hasClosedCaptions) {
+       leftTitle = localizedAudioTitle;
+       rightTitle = localizedSubtitlesTitle;
+    } else if (hasMultiAudioTracks) {
+      leftTitle = localizedAudioTitle;
+      rightTitle = "";
+    } else {
+      leftTitle = localizedSubtitlesTitle;
+      rightTitle = "";
+    }
+    
+    return (
+      <View style={styles.panelHeaderView}>
+        <View style={styles.panelHeaderViewLeftView}>
+          <Text style={[styles.panelHeaderViewLeftText]}>{leftTitle}</Text>
+        </View>
+        <View style={styles.panelHeaderViewRightView}>
+          <Text style={[styles.panelHeaderViewRightText]}>{rightTitle}</Text>
+          <TouchableHighlight style={styles.dismissButton}
+            accessible={true}
+            accessibilityLabel={BUTTON_NAMES.DISMISS} 
+            accessibilityComponentType="button"
+            underlayColor="transparent" // Can't move this property to json style file because it doesn't works
+            onPress={this.onDismissPress}>
+            <Text style={styles.dismissIcon}>
+              {this.props.config.icons.dismiss.fontString}
+            </Text>
+          </TouchableHighlight>
+        </View>
+      </View>
+    );
+  },
+
+  renderAudioSelectionScrollView: function() {
+    return (
+      <ItemSelectionScrollView 
+        style={styles.panelItemSelectionView}
+        items={this.props.audioTracksTitles}
+        selectedItem={this.props.selectedAudioTrackTitle}
+        onSelect={(item) => this.onAudioTrackSelected(item)}
+        config={this.props.config}>
+      </ItemSelectionScrollView>
+    );
+  },
+
+  renderCCSelectionScrollView: function() {
+    var selectedClosedCaptionsLanguage = this.props.selectedClosedCaptionsLanguage;
+    var offButtonTitle = Utils.localizedString(this.props.config.locale, "Off", this.props.config.localizableStrings);
+
+    if (!this.props.closedCaptionsLanguages || this.props.closedCaptionsLanguages[0] !== offButtonTitle) {
+      this.props.closedCaptionsLanguages.splice(0, 0, offButtonTitle)
+    }
+
+    if (!selectedClosedCaptionsLanguage || selectedClosedCaptionsLanguage == offButtonTitle || selectedClosedCaptionsLanguage == "") {
```

Comment:
use !! for cast string to bool. And use `===` to compare.

Reply:
"==" is better than "===" in this case (string)